### ISSUE TITLE
Media shortcut(s) not working on Mac

### ISSUE THREAD

**Issue by sarkipo** [EXMARaLDA/exmaralda#416]
Our student assistant has a Mac (will ask for details if needed) and a freshly installed Exmaralda, and the playback shortcut doesn't work for her (Ctrl+Space, also tried Cmd+Space).
(related to #113)

**Comment by berndmoos** [comment]
Yes, need details, at least the OS version. In any case, shortcuts are sometimes overriden by the machine's configuration, so the only way to tackle this is probably through configurable short-cuts via #113

**Comment by sarkipo** [comment]
Macbook Air 13" 2020, macOS 13.2
M1

**Comment by berndmoos** [comment]
I guess it should be "Cmd + Space" on the MAC, but that seems to be reserved by the system for Spotlight:

> Command–Space bar: Show or hide the [Spotlight](https://support.apple.com/guide/mac-help/mchlp1008/mac) search field. To perform a Spotlight search from a Finder window, press Command–Option–Space bar. (If you [use multiple input sources](https://support.apple.com/guide/mac-help/type-language-mac-input-sources-mchlp1406/mac) to type in different languages, these shortcuts change input sources instead of showing Spotlight. Learn how to [change a conflicting keyboard shortcut](https://support.apple.com/guide/mac-help/mchlp2864/mac).)

**Comment by sarkipo** [comment]
Yes, there was some Spotlight window appearing

**Comment by sarkipo** [comment]
> I guess it should be "Cmd + Space" on the MAC, but that seems to be reserved by the system for Spotlight:
> 
> > Command–Space bar: Show or hide the [Spotlight](https://support.apple.com/guide/mac-help/mchlp1008/mac) search field. To perform a Spotlight search from a Finder window, press Command–Option–Space bar. (If you [use multiple input sources](https://support.apple.com/guide/mac-help/type-language-mac-input-sources-mchlp1406/mac) to type in different languages, these shortcuts change input sources instead of showing Spotlight. Learn how to [change a conflicting keyboard shortcut](https://support.apple.com/guide/mac-help/mchlp2864/mac).)

So is the current way to change shortcuts on the Mac, not in PE?

**Comment by berndmoos** [comment]
> So is the current way to change shortcuts on the Mac, not in PE?

If you need a quick fix, yes.

**Comment by berndmoos** [comment]
Related to #113, so closing this one.
